_rules/SC4-1-1-aria-valid.md

```diff
@@ -0,0 +1,114 @@
+---
+name: Validity of ARIA attribute specified
+
+description: |
+ This rule checks that the `aria-*` attribute specified is valid.
+
+success_criterion:
+- 4.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any DOM element within the `body` of a webpage with an `aria-*` attribute.
```

Comment:
Why within the body? I would also be more specific about the `aria-*`, something like "with an attribute that starts with `aria-`".

Reply:
Note to self:

- Keep it generic, and say any HTML or SVG element (research)